docs/topics/ref_new-features-7-1-0.adoc

```diff
@@ -0,0 +1,104 @@
+====
+
+
+
+.Support for analyzing .NET applications has been added to MTA
+
+In {ProductFullName} 7.1.0, you can use {ProductShortName} to analyze Windows-only .NET framework applications to aid migration from version 4.5 or later to multi-platform .NET 8.0 running on OpenShift Container platform.
+
+[IMPORTANT]
+====
+Support for analyzing .NET applications is Developer Preview feature only. Developer Preview feature is not supported by Red{nbsp}Hat in any way and is not functionally complete or production-ready. Do not use Developer Preview feature for production or business-critical workloads. Developer Preview feature provides early access to upcoming product feature in advance of its possible inclusion in a Red{nbsp}Hat product offering. Customers can use this feature to test functionality and provide feedback during the development process. This feature might not have any documentation, is subject to change or removal at any time, and has received limited testing. Red{nbsp}Hat might provide ways to submit feedback on Developer Preview feature without an associated SLA.
+
+For more information about the support scope of Red{nbsp}Hat Developer Preview software, see link:https://access.redhat.com/support/offerings/devpreview/[Developer Preview Support Scope].
+====
+
+
+
+.Support for languages other than Java
+
+In {ProductFullName} 7.1.0, you can use {ProductShortName} to analyze applications written in languages other than Java. To run analysis on applications written in languages other than Java, add a custom rule set and do not specify a target language.
+
+[IMPORTANT]
+====
+Support for languages other than Java is Developer Preview feature only. Developer Preview feature is not supported by Red{nbsp}Hat in any way and is not functionally complete or production-ready. Do not use Developer Preview feature for production or business-critical workloads. Developer Preview feature provides early access to upcoming product feature in advance of its possible inclusion in a Red{nbsp}Hat product offering. Customers can use this feature to test functionality and provide feedback during the development process. This feature might not have any documentation, is subject to change or removal at any time, and has received limited testing. Red{nbsp}Hat might provide ways to submit feedback on Developer Preview feature without an associated SLA.
+
+For more information about the support scope of Red{nbsp}Hat Developer Preview software, see link:https://access.redhat.com/support/offerings/devpreview/[Developer Preview Support Scope].
+====
+
+
+
+.Assessment and review statuses are now displayed in the MTA UI
+
+Previously, the status of assessment and review was only displayed for applications. With this update, the status of the archetype assessment and review processes is displayed in the {ProductFullName} user interface (UI):
```

Comment:
```suggestion
In earlier releases of {ProductShortName}, the status of the assessment and the review was only displayed for applications. With this update, the status of the archetype assessment and review processes is displayed in the {ProductFullName} user interface (UI):
```

Reply:
Is it something that you used in other release notes docs? I'm just used to using "Previously" as we have a template for RHEL docs. But I have absolutely nothing against using your suggested warning. I'm ready to learn MTA docs rules :-)